index.html

```diff
@@ -24,17 +24,18 @@
   <body>
     <header>
       <nav id="toolbar">
-        <a href="#" id="logo">My Logo</a>
+        <a href="#" class="logo">My Logo</a>
         <div id="menu-bars-container">
-          <a href="#" id="hamberger"><img src="/images/Icon - Menu.svg" alt="hamburger icon"/></a>
-           <ul class="menu-bars">
-              <li class="mebu-bar">
+          <a href="#" ><img src="./images/ham-Icon.svg" alt="hamburger icon" class="hamberger"/></a>
+          <a href="#" class="close_container"><img src="./images/cancel-Icon.svg" alt="cancel icon" class="close-button invisible"></a>
+           <ul class="menu-list invisible">
+              <li class="menu-bar">
                 <a href="#works-container">Portfolio</a>
               </li>
-              <li class="mebu-bar">
+              <li class="menu-bar">
                 <a href="#about-container">About</a>
               </li>
-              <li class="mebu-bar">
+              <li class="menu-bar">
                 <a href="#footer-header">Contact</a>
```

Comment:
- Good job done so far 👏, but there are some differences between your project and the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Your project:
-- **OPTIONAL** - Please ensure your logo is centered in the middle of the navbar to help improve the UI of your website 👍 👍 
<img width="148" alt="bjejaet" src="https://user-images.githubusercontent.com/73607512/208875495-1d7f5052-89b5-448d-9cb1-6878e06b02d5.PNG">

-- You have not styled the mobile menu the way it is in the design template, please remember to stick to the styles in the design (fonts, color, bg images, margins, et.c..). Please style your mobile menu so it looks close to the one in the Figma template 🙏 
<img width="153" alt="egwrgwrg" src="https://user-images.githubusercontent.com/73607512/208875423-46c61882-22e6-4273-9fcd-fb53305ec80a.PNG">

Figma Design:
<img width="145" alt="bqehfbqhe" src="https://user-images.githubusercontent.com/73607512/208875456-bd25b148-2c6b-4ebb-a9f3-36655f989d54.PNG">